Footer

LivayaaSharma opened the pull request:
<img width="938" height="94" alt="Screenshot 2025-10-12 173153" src="https://github.com/user-attachments/assets/f7908fc2-57a7-4658-8ef1-3cf2caeea74a" />



I added a new components folder that includes the Footer.tsx file, along with an icons folder containing an index.tsx file to manage all icon imports in one place. I also created a public folder to hold static assets, right now it has the PSE logo and the Linktree icon.

The footer itself follows the layout from the Figma design and uses colors close to the mockup instead of predefined site colors, since there weren’t any set yet. I rendered the footer in layout.tsx so it appears across all pages.

All the social links except Discord are working because I couldn’t find the official Discord link yet but will update it once I have it. Also, should the PSE logo link back to the homepage or stay static for now? Let me know what you’d prefer.

If there are any design or spacing changes you’d like me to make, I’ll be happy to adjust!

swedenagu commented on src/components/Footer.tsx:
```diff
@@ -0,0 +1,72 @@
+                height={22}
+                className="object-contain"
+              />
+            </Link>
```
use the linktree icon from react-icons instead of an image

swedenagu commented on src/components/Footer.tsx:
```diff
@@ -0,0 +1,72 @@
+import Image from "next/image";
+import Link from "next/link";
+import { FaInstagram, FaLinkedin, FaDiscord } from "@/icons";
```
you don't necessarily need to have an index file if you're only using the icons in one place, you can import directly

LivayaaSharma commented:
<img width="936" height="91" alt="Screenshot 2025-10-18 194928" src="https://github.com/user-attachments/assets/77219f1b-3ad4-4e06-a0a7-87454d4173c1" />


Hy, so I updated the footer component based on feedback. I removed the Discord icon, switched the Linktree image to the official react-icons Linktree icon (SiLinktree from react-icons/si), and updated all the social links to the correct URLs provided.

The footer still uses the PSE Figma color palette and is rendered in layout.tsx so it appears across all pages. Also cleaned up the code by importing icons directly instead of using a separate index.tsx file.

Let me know if you’d like me to link the PSE logo to the homepage or make any other changes, happy to adjust! :)

brandontran1220 commented on src/components/Footer.tsx:
```diff
@@ -0,0 +1,63 @@
+        <div className="flex flex-col">
+          <div className="flex items-center gap-2">
+            <Image
+              src="/PSE-Gradient-Logo-3.webp"
```
```suggestion
              src={logo}
```

brandontran1220 commented on src/components/Footer.tsx:
```diff
@@ -0,0 +1,63 @@
+              height={24}
+              className="object-contain"
+            />
+            <h1 className="text-2xl font-semibold text-[#FFC653]">ΠΣΕ</h1>
```
```suggestion
            <p className="text-2xl font-semibold text-pse-yellow-100">ΠΣΕ</p>
```

brandontran1220 commented on src/components/Footer.tsx:
```diff
@@ -0,0 +1,64 @@
+              src={logo}
+              alt="PSE Logo"
+              width={32}
+              height={32}
```
```suggestion
              width={45}
              height={45}
```

brandontran1220 commented on src/components/Footer.tsx:
```diff
@@ -0,0 +1,64 @@
+              height={32}
+              className="object-contain"
+            />
+            <p className="text-pse-yellow-100 text-2xl font-semibold">ΠΣΕ</p>
```
```suggestion
            <p className="text-pse-yellow-100 text-5xl font-pse-gfs-didot font-medium">ΠΣΕ</p>
```

brandontran1220 commented on src/components/Footer.tsx:
```diff
@@ -0,0 +1,64 @@
+              target="_blank"
+              className="transition hover:opacity-80"
+            >
+              <SiLinktree size={24} />
```
```suggestion
              <SiLinktree size={35} />
```

brandontran1220 commented on src/components/Footer.tsx:
```diff
@@ -0,0 +1,64 @@
+
+          <Link
+            href="mailto:pres.pseatucr@gmail.com"
+            className="text-md hover:underline"
```
```suggestion
            className="text-xl hover:underline font-pse-gfs-didot"
```

brandontran1220 reviewed (changes requested):
I made the text and logo slightly bigger. I also added the correct font for the text! Just implement this and you're all set!

brandontran1220 reviewed (approved):
lgtm!
